Sources/Workspace/Workspace.swift

```diff
@@ -612,15 +613,16 @@ extension Workspace {
         createMultipleTestProducts: Bool = false,
         createREPLProduct: Bool = false,
         forceResolvedVersions: Bool = false,
+        shouldResolveMissingDepenencies: Bool = true,
```

Comment:
I think we should make this an enum because this should be mutually exclusive with `forceResolvedVersions`. Maybe something like:

```swift
enum ResolutionKind {
    case forceResolvedVersions
    case automaticResolution
    case none
}
```

Reply:
right. Makes sense here